src/activities/quizzes/QuizEntity.js

```diff
@@ -480,7 +480,8 @@ export class QuizEntity extends Entity {
 			[this.password(), quiz.password],
 			[this.notificationEmail(), quiz.notificationEmail],
 			[this.isPreventMovingBackwardsEnabled(), quiz.preventMovingBackwards],
-			[this.isAutoSetGradedEnabled(), quiz.autoSetGraded]
+			[this.isAutoSetGradedEnabled(), quiz.autoSetGraded],
+			[JSON.stringify(this.attempts()), JSON.stringify(quiz.attempts)]
```

Comment:
I might be wrong here (maybe @wongvincent can chip in) but I don't think we to include attempts here. This equals function is primarily for comparing properties on the quiz entity for the dirty check, but the actual attempts entity is separate while the quiz only has a simple attempts link.

Reply:
Yeah, since we'll have a separate siren-sdk entity for attempts and timing, we can have dirty checks in those entities instead.
Let's worry about that later.

Reply:
ok thanks, I will remove the attempts comparison from the equals function